cmd/controller/app/options/options.go

```diff
@@ -105,6 +105,11 @@ type ControllerOptions struct {
 	EnablePprof bool
 
 	DNS01CheckRetryPeriod time.Duration
+
+	// Annotations copied Certificate -> CertificateRequest,
+	// CertificateRequest -> Order. Slice of string literals that are
+	// treated as prefixes for annotation keys.
+	CopiedAnnotations []string
```

Comment:
What about Secrets, are we not also trying to cover those?

Reply:
Currently there are no annotations copied from Certificates to Secrets. I think for Secrets, the secret templates might make more sense if there's a need for custom annotations?